charts/seed-operatingsystemconfig/original/templates/kubelet/_kubelet.flags

```diff
@@ -15,6 +15,9 @@
 {{- if semverCompare "< 1.14" .Values.kubernetes.version }}
 --feature-gates=SupportPodPidsLimit=true
 {{- end }}
+{{- if .Values.enableContainerD }}
+--feature-gates=CRIContainerLogRotation=true
```

Comment:
Is this feature gate working for all Kubernetes version we support (1.10 - 1.17)?

Reply:
Yes,

Feature | Default | Stage | Since | Until
-- | -- | -- | -- | --
CRIContainerLogRotation | false | Alpha | 1.10 | 1.10
CRIContainerLogRotation | true | Beta | 1.11



CRIContainerLogRotation | false | Alpha | 1.10 | 1.10
CRIContainerLogRotation | true | Beta | 1.11